components/brave_wallet/browser/json_rpc_service_unittest.cc

```diff
@@ -4265,4 +4324,12 @@ TEST_F(ENSL2JsonRpcServiceUnitTest, GetEthAddr) {
   base::RunLoop().RunUntilIdle();
 }
 
+TEST_F(ENSL2JsonRpcServiceUnitTest, GetContentHash) {
```

Comment:
Shouldn't we have tests cover cases that would return errors? Same for GetEthAddr.

Reply:
added https://github.com/brave/brave-core/pull/14945/commits/a5bb9daf4cb54d02ea6c0f2213115c4cfc6fb581